node/package.json

```diff
@@ -12,23 +12,25 @@
     "ramda": "^0.25.0"
   },
   "devDependencies": {
-    "@types/atob": "^2.1.2",
-    "@types/bluebird": "^3.5.25",
-    "@types/co-body": "0.0.3",
-    "@types/jest": "^27.0.2",
-    "@types/jsonwebtoken": "^8.5.0",
+    "@types/co-body": "^0.0.3",
+    "@types/jest": "^24.0.18",
     "@types/node": "^12.0.0",
     "@types/ramda": "types/npm-ramda#dist",
     "@vtex/api": "6.45.15",
-    "@vtex/prettier-config": "^0.3.1",
-    "@vtex/tsconfig": "^0.6.0",
-    "tslint": "^5.12.0",
-    "tslint-config-prettier": "^1.18.0",
-    "tslint-config-vtex": "^2.1.0",
+    "@vtex/test-tools": "^1.0.0",
+    "@vtex/tsconfig": "^0.5.6",
+    "casual": "^1.6.2",
+    "ts-mockito": "^2.6.1",
     "typescript": "3.9.7",
     "vtex.storefront-permissions": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.storefront-permissions@1.33.2/public/@types/vtex.storefront-permissions"
   },
+  "resolutions": {
+    "@types/testing-library__dom": "6.12.1",
+    "@types/express": "4.16.0",
+    "@types/express-serve-static-core": "4.16.0"
+  },
   "scripts": {
-    "lint": "tsc --noEmit && tslint -c tslint.json './**/*.ts'"
+    "lint": "tsc --noEmit --pretty",
```

Comment:
Why remove tslint? Now this script only builds the code, but doesn't run lint anymore.

Reply:
Oops, I was fixing the issues with lint and I took the [template example](https://github.com/vtex-apps/service-example/blob/main/node/package.json#L22).

Reply:
Maybe tslint is already integrated into typescript itself, but I don't think so. Can you double check if when you run this it catches lint errors (maybe fake one and see if it's detected like this vs by calling tslint directly).

Reply:
@mairatma. Why is the lint configured different in root [`package.json`](https://github.com/vtex-apps/b2b-organizations-graphql/blob/master/package.json#L6) and node [`package.json`](https://github.com/vtex-apps/b2b-organizations-graphql/blob/master/node/package.json#L32)?

Reply:
Now it isn't failing, so I keep the previous config to decrease changes in this PR.